### PR TITLE
Feature/game complete dialog

### LINKED ISSUE #95: goes past the end of game
image

￼
￼

### LINKED ISSUE #106: add winners podium for game end
add dialog to show first, second and third place on end of game

### PR DESCRIPTION
**Pull request by areisle**
resolves #106 and resolves #95

### CODE REVIEW

File: src/components/GameCompleteDialog/index.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    display: grid;
+    grid-template-areas: 
+        '...... first .....'
+        'second first .....'
+        'second first third'
+        '...... first .....';
+    grid-template-columns: repeat(3, min-content);
+    grid-template-rows: repeat(3, min-content) 10px;
+
+    .podium {
```
Comment (creisle) [inline src/components/GameCompleteDialog/index.scss:11]: why didn't you use the inheritance here?

```scss

.podium {

  &--first {}
```

etc.

File: src/components/GameCompleteDialog/index.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        players,
+        scores,
+        stage,
+    } = props;
+
+    const [dismissed, setDismissed] = useState(false);
+
+    const winners = useMemo(() => getGameWinners(players, scores), [players, scores])
+
+    const gameComplete = stage === GAME_STAGE.COMPLETE || true;
```
Comment (creisle) [inline src/components/GameCompleteDialog/index.tsx]: leftover from testing?

File: src/icons.scss [hunk 3]
```diff
@@ -22,7 +22,18 @@
 }
 
 .trophy-icon {
-    fill: gold !important;
     font-size: 3em !important;
     stroke: black;
+
+    &--gold {
```
Comment (creisle) [inline src/icons.scss:28]: would you need important if you did multi-class?

```
& &--silver {}
```

**Review by creisle — CHANGES_REQUESTED** [review]

File: src/components/GameCompleteDialog/index.tsx (hunk 2 above)
Comment (areisle) [reply src/components/GameCompleteDialog/index.tsx]: ya https://github.com/areisle/quarantined_wizzards/pull/118/commits/0c5dc7682220787d6d19fdc60e6ceeefdcf86324

File: src/icons.scss (hunk 3 above)
Comment (areisle) [reply src/icons.scss:28]: ya I tried that as a fix already

File: src/components/GameCompleteDialog/index.scss (hunk 1 above)
Comment (areisle) [reply src/components/GameCompleteDialog/index.scss:11]: a0184c3cf3a272542dde6f7baa8ba5700c3aa291

**Review by creisle — APPROVED** [review]

**Pull request merged**